sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -177,7 +177,7 @@ The following section provides several code snippets covering some of the most c
 * [Recognize Receipts](#recognize-receipts "Recognize receipts")
 * [Recognize Business Cards](#recognize-business-cards "Recognize business cards")
 * [Recognize Invoices](#recognize-invoices "Recognize invoices")
-* [Recognize Id Documents](#recognize-id-documents "Recognize id documents")
+* [Recognize ID Documents](#recognize-id-documents "Recognize ID documents")
```

Comment:
We can remove the examples for all the individual prebuilts and just add one for `Using Prebuilt Models`

Reply:
I missed that. Thanks for catching it! Done.